docs/sources/trackers/google-tag-manager/previous-versions/v3/v3-tags/ecommerce-tag-template/index.md

```diff
@@ -3,13 +3,13 @@ title: "Snowplow v3 Ecommerce"
 sidebar_position: 2000
 ---
 
-This template implements the [Snowplow Ecommerce plugin](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/ecommerce/index.md) for the Snowplow JavaScript tracker v3 and is meant to be used alongside [the main Snowplow v3 tag template](/docs/sources/trackers/javascript-trackers/web-tracker/tracker-setup/google-tag-manager-custom-template/v3-tags/tag-template-guide/index.md).
+This template implements the [Snowplow Ecommerce plugin](/docs/sources/trackers/javascript-trackers/web-tracker/tracking-events/ecommerce/index.md) for the Snowplow JavaScript tracker v3 and is meant to be used alongside [the main Snowplow v3 tag template](/docs/sources/trackers/google-tag-manager/previous-versions/v3/v3-tags/tag-template-guide/index.md).
 
 ## Template Installation
 
 ### Tag Manager Gallery
 
-_Coming soon._
+The Ecommerce Tag Template can be found on the GTM Gallery [here](https://tagmanager.google.com/gallery/#/owners/snowplow/templates/snowplow-gtm-tag-template-ecommerce-v3).
```

Comment:
Nit: Although compliant with [G53](https://www.w3.org/WAI/WCAG22/Techniques/general/G53) I find this smells a bit like [F84](https://www.w3.org/WAI/WCAG22/Techniques/failures/F84.html) so I generally try to avoid link text like "here" and prefer things like:

> The Ecommerce Tag Template can be found on [the GTM Gallery](https://tagmanager.google.com/gallery/#/owners/snowplow/templates/snowplow-gtm-tag-template-ecommerce-v3).

Just personal preference, though, no need to change..